vnext/Microsoft.ReactNative/LayoutService.idl

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import "IReactContext.idl";
+
+#include "NamespaceRedirect.h"
+#include "DocString.h"
+
+namespace Microsoft.ReactNative
+{
+  [default_interface]
+  [webhosthidden]
+  DOC_STRING("Provides access to Yoga layout functionality.")
+  runtimeclass LayoutService {
+    DOC_STRING("Use this method to get access to the @LayoutService associated with the @IReactContext.")
+    static LayoutService FromContext(IReactContext context);
+
+    DOC_STRING("Applies layout to all root nodes.")
+    void ApplyLayout();
+
+    DOC_STRING("Applies layout from a given React node.")
+    void ApplyLayout(Int64 reactTag);
+
+    DOC_STRING("Applies layout from a given React node with root layout constraints.")
+    void ApplyLayout(Int64 reactTag, Single width, Single height);
+
+    DOC_STRING(
+      "Determines whether the UIManager is currently processing a batch of node updates."
+      "This is useful for optimizing layout and ensuring that applying layout to a particular "
+      "node will not cause tearing in the rendered UI.")
+    Boolean IsInBatch();
```

Comment:
this sounds like a get-only property?

Reply:
Good point. I can switch it to a property :)